### PR TITLE
P0   avatar & badge

### PR DESCRIPTION
**Pull request by saicharan-gouru**
Please review my avatar and badge components. I have attached the demos below.

Avatar component

Screenshot (175)

Badge component

badge

### CODE REVIEW

**Review by Creator-singh — APPROVED** [review]
looks good to go ahead🚀

**Review by ranitasahaa1230 — APPROVED** [review]
Looks good to me!

**Review by Mohasin-Haque — COMMENTED** [review]
Looks good.. everything is so cool.. But how about changing the classes like 
from small-avatar to avatar small.. Its in you BTW.. Great work😊

**Review by maitrakhatri — CHANGES_REQUESTED** [review]
in badge.css file, badge-on-text/ large-b... / button1 / badge-on-button /badge-on icon have many properties same such as position, width, font-size, padding, border-radius, color and font-weight. only the position dimensions are different. So instead of repeating the CSS, you can make a class and give all the elements same class

**Comment by saicharan-gouru** [comment]
> in badge.css file, badge-on-text/ large-b... / button1 / badge-on-button /badge-on icon have many properties same such as position, width, font-size, padding, border-radius, color and font-weight. only the position dimensions are different. So instead of repeating the CSS, you can make a class and give all the elements same class

Thanks for the review. I have made the changes.

**Comment by saicharan-gouru** [comment]
> Thanks for the review. I have made the changes.

Thanks for the review. I have made the changes.

File: badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    color: var(--white);
+    border-radius: 0.5rem;
+    border: 0px;
+}
+
+.badge-on-button {
+    top: -3.9rem;
+    left: -0.8rem;
+    width: max-content;
+    padding: 0.2rem 0.5rem;
```
Comment (shilpe26) [inline badge/badge.css:56]: consider using *px* instead of rem when value is < 1rem or 10px, as told by Vikram Sir

**Comment by arpit01923** [comment]
Good one

**Pull request merged**